Twilio/TwiML/GenericNode.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Twilio\TwiML;
+
+class GenericNode extends TwiML {
+
+	/**
+	 * GenericNode constructor.
+	 *
+	 * @param string $name XML element name
+	 * @param string $value XML value
+	 * @param array $attributes XML attributes
+	 */
+	public function __construct($name, $value = null, $attributes = array()) {
```

Comment:
I would drop the default values for `$value` and `$attributes`, since this should only ever be called from `TwiML::addChild`